autospec/specfiles.py

```diff
@@ -109,6 +139,7 @@ def write_spec(self):
         self.write_lang_files()
 
         self.specfile.close()
+        return "generate"
```

Comment:
Does "generate" mean continue with today's behavior of autospec -- generating the specfile from scratch?

Reply:
Yep, open to other words/handling. I'm just treating it as an enum basically.